db/migrations/20220919123300-test-db-migration.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
```

Comment:
Do we still need this file?

Reply:
Not sure, it is for demo/once off verification purpose.

Reply:
ah right